pgserviceparser/__init__.py

```diff
@@ -12,12 +13,16 @@ def conf_path() -> Path:
     """
     pg_config_path = None
     if getenv("PGSERVICEFILE"):
-        pg_config_path = getenv("PGSERVICEFILE")
+        pg_config_path = Path(getenv("PGSERVICEFILE"))
     elif getenv("PGSYSCONFDIR"):
-        pg_config_path = path.join(getenv("PGSYSCONFDIR"), "pg_service.conf")
+        pg_config_path = Path(getenv("PGSYSCONFDIR")) / "pg_service.conf"
     else:
-        pg_config_path = "~/.pg_service.conf"
-    return Path(pg_config_path).expanduser()
+        if platform.system() == "Windows":
+            pg_config_path = Path(getenv("APPDATA")) / "postgresql/.pg_service.conf"
```

Comment:
Should this use should this better use https://docs.python.org/3/library/os.path.html#os.path.expandvars ?